src/apps/dialogs/SkillConfigDialog.svelte

```diff
@@ -13,16 +13,47 @@
 
     export let { document, appId, skillKey } = getContext("#external").application;
 
+    function getSkillSpecialties(skillKey) {
+	let skillSpecialties = CONFIG.A5E.skillSpecialties[skillKey];
+
+	if (game.settings.get("a5e", "hideVRCSpecialties")) {
+		if (skillKey == "arc") {
+			delete skillSpecialties.psionics;
+			delete skillSpecialties.psionicItems;
+			delete skillSpecialties.psionicCreatures;
+		} else if (skillKey == "ath") {
+			delete skillSpecialties.zeroG;
+		} else if (skillKey == "eng") {
+			delete skillSpecialties.robotics;
+			delete skillSpecialties.starships;
```

Comment:
This should really be handled in the config. Currently you're deleting them from the config on skillconfig dialog load.

Reply:
Completed.